core/templates/components/ck-editor-helpers/ck-editor-copy-content-service.ts

```diff
@@ -70,8 +74,11 @@ export class CkEditorCopyContentService {
         break;
       }
 
-      if (currentElement.parentElement.tagName === this.OUTPUT_VIEW_TAG_NAME) {
+      if (currentElement.parentElement.tagName === this.OUTPUT_VIEW_TAG_NAME ||
+        currentElement.tagName === this.OUTPUT_VIEW_TAG_NAME) {
         break;
+        // Prevents errors if the element clicked on is itself the
```

Comment:
The error reads "Cannot read tagName of null" since the super element defined by ANGULAR_HTML_BIND does not have a parent element (it is the parent element for everything included inside it).

Reply:
Should I specify this as a comment?